frontend/src/utils/tsxUtils.tsx

```diff
@@ -394,25 +393,30 @@ export class StatusIndicator extends Component<StatusIndicatorProps> {
     }
 
     customRender() {
-        const content = <div style={{ marginBottom: '0.2em' }} className={this.showWaitingText ? 'waitingForMessagesBox waitingForMessagesText' : ''}>
-            <div style={StatusIndicator.progressStyle}>
-                <Progress percent={this.props.fillFactor * 100} showInfo={false} status="active" size="small" style={{ lineHeight: 1 }} />
-            </div>
-            <div style={StatusIndicator.statusBarStyle}>
-                <div>{this.showWaitingText ? 'Kafka is waiting for new messages...' : this.props.statusText}</div>
-                <div style={StatusIndicator.progressTextStyle}>{this.props.progressText}</div>
-            </div>
-            {(this.props.bytesConsumed && this.props.messagesConsumed) &&
-                <div style={StatusIndicator.statusBarStyle}>
-                    <div style={{ display: 'flex', alignItems: 'center' }}>
-                        <DownloadOutlined style={{ color: colors.brandOrange }} /> {this.props.bytesConsumed}
-                    </div>
-                    <div style={{ display: 'flex', alignItems: 'center', marginLeft: 'auto' }}>
-                        <CopyOutlined style={{ color: colors.brandOrange }} />{this.props.messagesConsumed} messages
+        const content =
+            <ChakraProvider theme={redpandaTheme}>
+                <div style={{marginBottom: '0.2em'}}>
+                    <Progress
+                        value={this.props.fillFactor * 100}
+                        isIndeterminate={this.props.statusText === 'Connecting'}
+                        colorScheme="brand"
+                    />
+                    <div style={StatusIndicator.statusBarStyle}>
+                        <div>{this.showWaitingText ? 'Kafka is waiting for new messages...' : this.props.statusText}</div>
+                        <div style={StatusIndicator.progressTextStyle}>{this.props.progressText}</div>
                     </div>
+                    {(this.props.bytesConsumed && this.props.messagesConsumed) &&
+                        <div style={StatusIndicator.statusBarStyle}>
+                            <div style={{display: 'flex', alignItems: 'center'}}>
+                                <DownloadOutlined style={{color: colors.brandOrange}}/> {this.props.bytesConsumed}
+                            </div>
+                            <div style={{display: 'flex', alignItems: 'center', marginLeft: 'auto'}}>
+                                <CopyOutlined style={{color: colors.brandOrange}}/>{this.props.messagesConsumed} messages
+                            </div>
+                        </div>
+                    }
                 </div>
-            }
-        </div>
+            </ChakraProvider>
```

Comment:
A few questions:

- When migrating this file to use Chakra UI, we have added another `ChakraProvider` component. Would it be possible to avoid it, given we would have several instances of Chakra provider, and it might be injecting CSS several times, I am a bit worried (I know this works, but maybe we don't need the nesting as much)
- Should the convention be to also migrate the remaining parts in this file, for example
```tsx
<div style={{display: 'flex', alignItems: 'center'}}>
```
becomes
```tsx
<Flex alignItems="center">
```

or
```tsx
<div style={StatusIndicator.statusBarStyle}>
```
becomes
```tsx
<Flex fontWeight="bold"> // The rest of the properties should be in a global config
```

We can also do it in separate PRs, but just wondering what the convention/strategy should be, to ensure that by the end after we migrated all components we are using Chakra in the designed way

Reply:
@malinskibeniamin thanks for your comment and a note about the convention to convert styles. I'll do that from now on and I'll do that as part of this PR.

When it comes to the ChakraProvider, it's needed here because we're in a modal and thanks to `customRender` method the existing ChakraProvider doesn't work. I think this will no longer be an issue once we migrate modals to Chakra. Or do you have some other suggestion how to solve this "modal" problem?

Reply:
Good point, I think with modals it was an issue before, indeed once we have moved everything over including `Chakra` modals then hopefully the nested providers can go away.

Reply:
@malinskibeniamin I'm a bit new to Chakra, so the fastest will be to ask. How to move the `fontFamily` and `fontSize` to global config? I guess these two specific props are relevant to this local component only right?


 `<Flex fontWeight="bold" fontSize="80%" fontFamily='"Open Sans", sans-serif'>`

Reply:
I don't think we should change the fontFamily only for this 1 particular component, why not keep using `Inter` like we do elsewhere?

You could use it in the theme.ts under `fonts`

Reply:
@malinskibeniamin I wanted to reduce changes to the UI as much as possible. But I guess the consistency is more important. I'll change this once Progress component in merged in the UI

Reply:
@malinskibeniamin I think we can merge